src/app/talent/talent/talent.component.ts

```diff
@@ -106,11 +108,25 @@ export class TalentComponent extends AbstractComponent<Talent> {
 		});	
 	}
 
+	private togglePublished(index) {
+		this.medias[index].published = !this.medias[index].published;
```

Comment:
and here you can receive the object directly and code like:
item.published = !item.published